### PR TITLE
PR: Add release instructions on how to update the Conda-forge `rc` feedstock channel (Installers)

### PR DESCRIPTION
**Pull request by ccordoba12**
## Description of Changes

This describes how to update that channel with the changes in the `main` feedstock branch.

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: @ccordoba12

### CODE REVIEW

**Comment by ccordoba12** [comment]
@dalthviz, please take a look at these new instructions and let me know what you think.

**Comment by ccordoba12** [comment]
BTW, I opened https://github.com/conda-forge/spyder-feedstock/pull/206 by following the initial part of the instructions. Once 6.0.3rc1 is ready to be released in Conda-forge, you can follow the last steps in the same PR.

File: RELEASE.md [hunk 1]
```diff
@@ -182,6 +182,37 @@ For that you need to run the following commands:
...
+
+                  git checkout main
+                  git fetch upstream
+                  git merge upstream/main
+                  git diff main rc > update.patch
+                  patch -p1 -R < update.patch
+
+            - Revert changes to every file outside the `recipe` directory, except for `conda-forge.yml`. They will be applied when re-rendering the feedstock.
+
+            - Restore `recipe/conda_build_config.yaml` (that file contains the info to use a different channel).
```
Comment (dalthviz) [inline RELEASE.md:204]: Not completly sure of the syntax but maybe using something like

```
git diff main rc recipe/ ":!recipe/conda_build_config.yaml" > recipe.patch

``` 

To get the patch for the recipe dir contents and

```
git diff main rc conda-forge.yml > conda-forge.patch
```

to get the patch of the `conda-forge.yml` file?

Maybe it is not possible to mark in a single command the paths to ignore so the patch(s) can be applied without worrying about remove or leaving things @ccordoba12 ?

Comment (ccordoba12) [reply RELEASE.md:204]: Good idea @dalthviz! I didn't know you could include certain directories and exclude files from the diff.

I'll update the instructions accordingly.

Comment (ccordoba12) [reply RELEASE.md]: @dalthviz, please check again the simplified instructions.

**Review by dalthviz — APPROVED** [review]
Thanks @ccordoba12 !

**Comment by dalthviz** [comment]
@meeseeksdev please backport to 6.x

**Pull request merged**